animation server: `'ServerGoalHandle' object has no attribute 'aborted'`

```
[animation_node-14] [ERROR] [1673863108.856204582] [animation.action_server]: Error raised in execute callback: 'ServerGoalHandle' object has no attribute 'aborted'
[animation_node-14] Traceback (most recent call last):
[animation_node-14]   File "/opt/ros/rolling/local/lib/python3.10/dist-packages/rclpy/action/server.py", line 325, in _execute_goal
[animation_node-14]     execute_result = await await_or_execute(execute_callback, goal_handle)
[animation_node-14]   File "/opt/ros/rolling/local/lib/python3.10/dist-packages/rclpy/executors.py", line 108, in await_or_execute
[animation_node-14]     return callback(*args)
[animation_node-14]   File "/colcon_ws/install/bitbots_animation_server/lib/python3.10/site-packages/bitbots_animation_server/animation_node.py", line 88, in execute_cb
[animation_node-14]     goal.aborted(text="HCM not controllable. Will now become controllable. Try again later.")
[animation_node-14] AttributeError: 'ServerGoalHandle' object has no attribute 'aborted'
```

--- a/bitbots_animation_server/bitbots_animation_server/animation_node.py
+++ b/bitbots_animation_server/bitbots_animation_server/animation_node.py
@@ -85,7 +85,7 @@ class AnimationNode(Node):
             # but we send a request, so that we may can soon
             self.send_animation_request()
             self.get_logger().info("HCM not controllable. Only sent request to make it come controllable.")
-            goal.aborted(text="HCM not controllable. Will now become controllable. Try again later.")
+            goal.abort()
             return PlayAnimation.Result(successful=False)
 
         animator = self.get_animation_splines(self.current_animation, goal)

